episodes/filters-and-thresholding.md

```diff
@@ -0,0 +1,743 @@
+are trying to segment. It's worth trying a variety of options on your images to 
+see which performs best for your specific dataset. 
+
+:::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+## The Napari assistant
+
+In this episode, we have directly applied methods from the `Tools` menu to our 
+images. It's worth mentioning though that these methods are also available via 
+the 'Napari assistant'.
+
+First, let's clean up our layer list again. Make sure you only have the 'nuclei' 
+layer in the layer list - select any others and remove them by clicking the ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/delete.svg
+){alt="A screenshot of Napari's delete layer button" height='30px'} icon. Also, 
+close all settings panels on the right side of Napari by clicking the tiny ![](
+https://raw.githubusercontent.com/napari/napari/main/napari/resources/icons/visibility_off.svg
+){alt="A screenshot of Napari's hide button" height='20px'} icon at their top 
+left corner.
+
+We can open the assistant with:  
+`Tools > Utilities > Assistant (na)`
+
+![](fig/napari-assistant.png){alt="Screenshot of the Napari assistant user 
+interface"}
+
+The assistant splits image processing operations into broad categories such as 
+'Remove noise' and 'Filter'. Let's recreate our workflow of a gaussian blur 
+followed by otsu thresholding in the assistant. 
+
+First, click the 'Remove noise' button in the assistant panel. In the settings 
+that appear below, select the 'Operation' as 'Gaussian (scikit-image, nsbatwm)' 
+and set the 'sigma' to three. 
+
+![](fig/remove-noise-options.png){alt="Screenshot of settings for removing noise 
+in Napari"}
+
+This will create a new layer called 'Result of Gaussian (scikit-image, nsbatwm)'. 
+If you can't see it in the viewer, try adjusting the contrast limits so the 
+right node is at 1 (remember this is a float image that runs from 0-1, as we 
+discussed in an [earlier section](#thresholding-the-blurred-image)).
+
+With the 'Result of Gaussian (scikit-image, nsbatwm)' layer selected, click the 
+'Binarize' button in the assistant panel. Change the 'Operation' to 
+'Threshold (Otsu et al 1979, scikit-image, nsbatwm)'. 
+
+![](fig/binarize-options.png){alt="Screenshot of settings for binarize in 
+Napari"}
+
+You should see a nuclei mask similar to the one we created before. 
+
+Note that you can change the settings of a particular operation and all the 
+steps after it will also update. For example, select the 'Result of Gaussian 
+(scikit-image, nsbatwm)' layer again, then change the sigma to 1. You should see 
+the mask automatically updates, becoming more patchy and noisy.
+
+The napari assistant also has useful options to export and save workflows for 
+re-use. You can read more about the different options in the 
+[online documentation](https://www.napari-hub.org/plugins/napari-assistant).
```

Comment:
Here or somewhere else I think there might be something important to say about workflows. If you develop it on one image, then you may be tempted to change it for a second, but the workflow needs to be generalizable, i.e. you'd rather have a couple of small errors but in general it works well across all of your data. Just like risks to overfitting a statistical model.

Reply:
I'm not sure how to fit this point into the text of this episode - perhaps it's something to bring up in the 'Instance segmentation and measurements' episode where the napari-assistant is used more for a longer workflow? @davecash75 @thompson318